persons.rb

```diff
@@ -7,9 +7,14 @@ def initialize(age, name)
     super()
     @age = age
     @name = name
+    @rentals = []
   end
 
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Please modify your method to include the ability to create a rental. It should look something like this 
```
  def add_rental(book, date)
    rental = Rental.new(date, book, self)
    @rentals.push(rental)
  end
```